erpnext/shopping_cart/doctype/shopping_cart_settings/shopping_cart_settings.json

```diff
@@ -166,13 +167,20 @@
    "fieldname": "enable_variants",
    "fieldtype": "Check",
    "label": "Enable Variants"
+  },
+  {
+   "default": "0",
+   "depends_on": "eval: doc.enable_checkout == 0",
+   "fieldname": "draft_quotation",
+   "fieldtype": "Check",
+   "label": "Draft Quotation"
```

Comment:
"Draft Quotation" is not a proper label. Write a more descriptive label to the user. Also add description what enabling and disabling this check will do. Since this is dependent on enable checkout it should be in the checkout section, display settings is not the appropriate place